lpa-submissions-e2e-tests/cypress/support/commands.js

```diff
@@ -170,109 +157,17 @@ Cypress.Commands.add(
   require('./appeals-questionnaire-navigation/verifyCannotStartStatus'),
 );
 
-Cypress.Commands.add('verifyAppealDetailsSidebar', require('./common/verifyAppealDetailsSidebar'));
-
-Cypress.Commands.add('getAppealDetailsSidebar', require('./common/getAppealDetailsSidebar'));
-
-Cypress.Commands.add(
-  'inputAppealsReferenceNumber',
-  require('./appeals-in-immediate-area/inputAppealsReferenceNumber'),
-);
-
 Cypress.Commands.add(
-  'verifyAppealsSelectionYesHelpText',
-  require('./appeals-in-immediate-area/verifyAppealsSelectionYesHelpText'),
+  'goToTellUsAboutAppealsInImmediateAreaPage',
```

Comment:
Do we still need this go to page method? Managed to remove many of the others?

Reply:
Removed.